python/nano/src/bigdl/nano/deps/onnxruntime/onnxruntime_api.py

```diff
@@ -15,7 +15,8 @@
 #
 
 
-def PytorchONNXRuntimeModel(model, input_sample=None, onnxruntime_session_options=None):
+def PytorchONNXRuntimeModel(model, input_sample=None,
+                            onnxruntime_session_options=None, **kwargs):
     """
         Create a ONNX Runtime model from pytorch.
 
```

Comment:
add some comment to tell future developers what may be contained in kwargs?

Reply:
same things to some of other changes as well

Reply:
fix